lib/wrap-guide-element.coffee

```diff
@@ -44,25 +45,53 @@ class WrapGuideElement
 
   handleConfigEvents: ->
     updateGuideCallback = => @updateGuide()
+
+    updatePreferredLineLengthCallback = (args) =>
+      # ensure that the right-most wrap guide is the preferredLineLength
+      multiGuides = atom.config.get('wrap-guide.guides', scope: @editor.getRootScopeDescriptor())
+      if multiGuides.length > 0
+        multiGuides[multiGuides.length - 1] = args.newValue
+        multiGuides = uniq(i for i in multiGuides when i <= args.newValue)
+        atom.config.set 'wrap-guide.guides', multiGuides,
+          scopeSelector: ".#{@editor.getGrammar().scopeName}"
+      @updateGuide()
+
+    updateGuidesCallback = (args) =>
+      # ensure that multiple guides stay sorted in ascending order
+      guides = uniq(args.newValue)
```

Comment:
To avoid bringing in a new dependency, how about the following instead?
```coffee
guides = args.newValue.filter((item, index) -> args.newValue.indexOf(item) is index)
```
It's unclear to me if `uniq` sorts the array.  Nevertheless, that can also be done easily: `guides.sort((a, b) -> a - b)`

Reply:
Yep, makes sense to me. I wasn't totally sure if we wanted to bring in the dependency or just roll our own solution here. PR updated. I also made the function use more clear by naming it `uniqueAscending()`.